src/template.yml

```diff
@@ -68,21 +68,22 @@ Resources:
               StringEquals:
                 aws:PrincipalOrgID: !GetAtt Organization.OrganizationId
             Resource:
-              - !Sub arn:aws:s3:::${BootstrapTemplatesBucket}
-              - !Sub arn:aws:s3:::${BootstrapTemplatesBucket}/*
+              - !Ref BootstrapTemplatesBucket
```

Comment:
According to the [S3 Policy docs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-arn-format.html), the resource should refer to the Arn of a bucket in S3 Bucket Policies. When you use Ref, according to the [S3 CFn docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket.html#aws-properties-s3-bucket-return-values), it returns the bucket name instead.

I think this line should become:
```yaml
!GetAtt BootstrapTemplatesBucket.Arn
# Or:
!Sub "${BootstrapTemplatesBucket.Arn}"
```